source/_integrations/trend.markdown

```diff
@@ -106,6 +106,12 @@ sensors:
       required: false
       type: integer
       default: 0
+    unique_id:
```

Comment:
is this ID generated? Or do I create my own?

Reply:
You create your own; it is the same field as added to many other helpers.